Refactor config.php (#158)
DenverCoder1/github-readme-streak-stats: README.md

1. Visit [this link](https://github.com/settings/tokens/new?description=GitHub%20Readme%20Streak%20Stats) to create a new Personal Access Token
2. Scroll to the bottom and click **"Generate token"**

> 3. **Copy and paste** `.env.example` in the root `.env` directory and replace `TOKEN=` with **your token** and `USERNAME=` with **your username**:

Reviewer: ```suggestion
3. **Make a copy** of `.env.example` named `.env` in the root directory and add **your token** after `TOKEN=` and **your username** after `USERNAME=`
```

1. Visit [this link](https://github.com/settings/tokens/new?description=GitHub%20Readme%20Streak%20Stats) to create a new Personal Access Token
2. Scroll to the bottom and click **"Generate token"**

> 3. **Make a copy** of `.env.example` named `.env` in the root directory and **your token** after `TOKEN=` and **your username** after `USERNAME=`
> 

Reviewer: Forgot the word "add"...

```suggestion
3. **Make a copy** of `.env.example` named `.env` in the root directory and add **your token** after `TOKEN=` and **your username** after `USERNAME=`:
```
